XiEditor/LineCache.swift

```diff
@@ -92,7 +82,7 @@ class LineCache {
         }
         return nil
     }
-
+    
```

Comment:
I try to keep trailing space low, but XCode really seems to want to add it, doesn't it?